docusaurus/docs/recipes/dropdown.md

```diff
@@ -0,0 +1,427 @@
+---
+id: dropdown
+title: Implementing a Dropdown
+sidebar_label: Dropdown
+---
+
+Implementing a Dropdown/Select component from the scratch is always a big challenge. A lot of things have to be considered being some of them accessibility, autocomplete feature, multi-selection feature, native support (for touch devices mainly).
```

Comment:
```suggestion
Implementing a Dropdown/Select component from the scratch is always a big challenge with a lot of things to consider such as accessibility, autocomplete feature, multi-selection feature, native support (for touch devices mainly).
```

Reply:
I'd also change `from the scratch` to `from scratch` as it's more widely used.

Reply:
I missed that one but I updated my comment with that change now, thanks!